tests/conftest.py

```diff
@@ -431,6 +433,56 @@ def test_run_parameters():
     )
 
 
+@pytest.fixture(scope="function")
+def test_drivers():
+    drivers = ParameterSet()
+    setters = {
+        "start_time": np.datetime64("1810-03-04"),
+        "stop_time": np.datetime64("2450-06-01"),
+        "period_length": np.timedelta64(5000, "D"),
+        "emissions_parameter_type": ParameterType.AVERAGE_TIMESERIES,
+        "Equilibrium Climate Sensitivity": 2.5 * _unit_registry("delta_degC"),
+        "Radiative Forcing 2xCO2": 3.5 * _unit_registry("W/m^2"),
+    }
+
+    drivers.scalar(
+        ("Equilibrium Climate Sensitivity",), "K", region=("World",)
+    ).value = setters["Equilibrium Climate Sensitivity"].magnitude
+    drivers.scalar(
+        ("Radiative Forcing 2xCO2",), "W / m^2", region=("World",)
```

Comment:
```suggestion
        ("Radiative Forcing Sensitivity",), "W / m^2", region=("World",)
```

Reply:
Bad idea, `Radiative Forcing Sensitivity` is too vague (you also have e.g. concepts like `Radiative Forcing 2xCH4`)

Reply:
Hmm, then we chould be consistent with `Equilibrium Climate Sensitivity`...

Reply:
Have the specific gas (without "2x") in the lower level, i.e. `("Radiative Forcing Sensitivity", "CH43")`?

Reply:
> Hmm, then we chould be consistent with `Equilibrium Climate Sensitivity`

No they're unrelated concepts. This language matches the standards in the SCM (and wider climate modelling) community.

Reply:
> Radiative Forcing Sensitivity

As a concept, this makes no sense

Reply:
Of course they are totally different concepts (thats why it's not caleld "Temperature Sensitivity"), but how else would you call it? "Response" intead of "Sensitivity"?

Reply:
Like I've called it, following the convention in the community

Reply:
Hmm, that's not a proper name, just an identifier. Then just `("Radiative Forcing", "CH4")`?

Reply:
How do you mean not a proper name? 'Equilibrium Climate Sensitivity' is a quantity with a precise definition, as is 'Radiative Forcing 2xCO2', do you want it spelt out even more like 'Radiative Forcing due to Doubling of Atmospheric CO2 Concentrations'?